eras/shelley/impl/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Version history for `cardano-ledger-shelley`
 
-## 1.8.0.1
+## 1.9.0.0
```

Comment:
Perfect! This is exactly how it was suppose to be done for `cardano-ledger-babbage` changelog as well